contrib/kafka/config/500-webhook.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-webhook
+  namespace: knative-eventing
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: kafka-webhook
+      role: kafka-webhook
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels: *labels
+    spec:
+      serviceAccountName: kafka-webhook
+      containers:
+      - name: kafka-webhook
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: github.com/knative/eventing/contrib/kafka/cmd/webhook
```

Comment:
I wonder.... if there could be a more generic solution, instead of adding more and more new images for webhooks for different channels :thinking:

Reply:
Yeah, I was asking the same around here. And apparently there is none. When openAPI validation allows to set defaults in the yaml, we might be able to start using that. 
The other option is to allow an "invalid" kafka channel to be created, and then in the controller, set the defaults to 1 for example, if not specified. 
I think I like this webhook option better, that is why I went for this path.

Reply:
Yeah, I'm fine for pushing this in as a seprate binary for now, but agree on having fewer binaries to maintain. The tricky bit is to dynamically dispatch them. I think it might make sense to deploy a knative service for these, so that they would scale to 0 and we wouldn't have to have binaries running all the time for these. Thoughts? Perhaps we should open up an issue esp, once we start seeing more channels?

Reply:
Agree. Let me add an issue for that and we can tackle it in a follow up

Reply:
https://github.com/knative/eventing/issues/1325